### PR TITLE
Add fixes to language after copyediting

### PR DESCRIPTION
**Pull request by tarrow**
Rebased from the master branch of Jens' fork

### CODE REVIEW

**Review by manicki — APPROVED** [review]

File: src/components/data/lang/en.json [hunk 1]
```diff
@@ -1,19 +1,19 @@
 {
     "key": "en",
-    "title": "Blubber",
-    "subtitle": "A docker-compose.yml builder for wikibase-docker",
+    "title": "wikibase-docker Config Wizard",
```
Comment (addshore) [inline src/components/data/lang/en.json]: I'm almost against calling it this.
It makes it sound like this is the only way you can use the docker images.
At the end of the day this only generates a docker-composer file...

File: src/components/data/lang/en.json [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+    "subtitle": "A yaml builder for wikibase-docker",
```
Comment (addshore) [inline src/components/data/lang/en.json]: again, this should really be docker-compose specific.
yaml builder? what does that even mean? yaml + docker = nothing

File: src/components/data/lang/en.json [hunk 3]
```diff
@@ -27,38 +27,38 @@
...
-    "databaseUserPassword": "What should the database user password be?",
+    "databaseConfiguration": "Database (MySQL)",
+    "databasePort": "Which port to use for the database?",
+    "databaseName": "How would you like to name the database?",
+    "databaseUser": "What is the name of your database user?",
+    "databaseUserPassword": "Provide your database user with a password.",
 
-    "finish": "Download docker-compose.yml file",
-    "finish_description": "<p>You can download your docker-compose.yml file using the button to the right.</p><p>You can now run your Wikibase docker-compose environment.</p><p>To learn how to do this, refer to the <a target=\"_blank\" href=\"https://github.com/wmde/wikibase-docker/blob/master/README-compose.md\">manual</a></p>"
+    "finish": "Download Docker Compose file",
```
Comment (addshore) [inline src/components/data/lang/en.json]: See, here you say docker compose file, not wikibase-docker yml :/

**Review by addshore — CHANGES_REQUESTED** [review]
Some thoughts, I support most of it, and the fact that we want to change the name, but I think we need to make sure people dont think this is the only want to use wikibase and docker, and that the yml being generates is only for docker-compose

**Comment by tarrow** [comment]
@johl Can you respond to @addshore 's thoughts?

File: src/components/data/lang/en.json (hunk 2 above)
Comment (bitPogo) [reply src/components/data/lang/en.json]: We could also have no subtitle, I guess.

File: src/components/data/lang/en.json (hunk 3 above)
Comment (bitPogo) [reply src/components/data/lang/en.json]: Sure. So you want to skip the description?

**Comment by addshore** [comment]
Poke @johl

**Comment by manicki** [comment]
rebased the PR branch against the master. @addshore your concerns should now be no longer relevant, given the tool has been renamed in the other PR.

**Pull request merged**